### ISSUE TITLE
Removing unused Text import breaks compiled elm.js

### ISSUE THREAD

**Issue by kasbah** [elm-lang/elm-compiler#864]
[This commit](https://github.com/kasbah/mooltipass.hid-app/commit/b7226e969f587c1e5086028fe922c91a5e0826fa) that removes the (unused) Text module from my application compiles fine with elm-make but the resulting `elm.js` encounters this error. This is with `elm-compiler` 0.14. Tested on Chrome unstable, stable and Firefox (Linux 32-bit).

```
Uncaught TypeError: Cannot read property 'make' of undefined
Elm.Native.Graphics.Input.make elm.js:3400
Elm.Graphics.Input.make        elm.js:1553


Elm.Main.make                  elm.js:1814
init                           elm.js:4404
Elm.fullscreen                 elm.js:4292
load-elm.js:1(anonymous function)
```

[Compiled elm.js](https://gist.github.com/kasbah/d9d32e9f7291f46ef2e3)

**Comment by TheSeamau5** [comment]
I'm adding the findings documented in [this issue on elm-lang/core](https://github.com/elm-lang/core/issues/203) to this issue. I've closed the issue on elm-lang/core due to the actual cause being the problems described by Evan with `elm-make`.

**Comment by mgold** [comment]
Maybe it should always keep native modules?

**Comment by evancz** [comment]
This should be fixed in 0.15.1 so I am going to close. Thanks for the report!

I think we will have a different way of sorting out which native modules are needed once we have dead code elimination going, so I think that'll bring a nice solution here.
